migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala

```diff
@@ -139,7 +139,14 @@ final case class JournalMigrator(profile: JdbcProfile)(implicit system: ActorSys
       .forceInsert(journalSerializedRow)
 
     val tagInserts =
-      newJournalQueries.TagTable ++= tags.map(tag => TagRow(journalSerializedRow.ordering, tag)).toSeq
+      newJournalQueries.TagTable ++= tags
+        .map(tag =>
+          TagRow(
+            Some(journalSerializedRow.ordering),
+            Some(journalSerializedRow.persistenceId),
+            Some(journalSerializedRow.sequenceNumber),
+            tag))
```

Comment:
This migration tool is intended to migrate from the previous schema (without `event_tag` table) to the current one with the `event_tag` table. 

So in that sense, we could set the ordering to `None` or use the redundant flag to decide whether we want the order.

Reply:
I have noticed that when verifying my tests, there doesn't seem to be much of a difference between them. Therefore, I did not make any changes.